service/controller/resource/alert/rules/labellingschema.go

```diff
@@ -40,11 +40,8 @@ func LabellingSchemaValidationRule(obj interface{}) (metav1.Object, error) {
 					Rules: []promv1.Rule{
 						promv1.Rule{
 							Alert: "InvalidLabellingSchema",
-							Expr: intstr.IntOrString{
-								Type:   intstr.String,
-								StrVal: `up{cluster_type!~"control_plane|tenant_cluster"} or up{app=""} or up{installation=""} or up{cluster_id=""} or up{instance!~"(\\d+\\.\\d+\\.\\d+\\.\\d+:\\d+)||vault.*"} or up{provider!~"aws|azure|kvm"}`,
-							},
-							For: "10m",
+							Expr:  intstr.FromString(`up{cluster_type!~"control_plane|tenant_cluster"} or up{app=""} or up{installation=""} or up{cluster_id=""} or up{instance!~"(\\d+\\.\\d+\\.\\d+\\.\\d+:\\d+)||vault.*"} or up{provider!~"aws|azure|kvm"}`),
```

Comment:
Is this related?

Reply:
No, I just discovered the intstr.FromString util function during this PR